csrc/quantization/cutlass_w8a8/broadcast_load_epilogue_c2x.hpp

```diff
@@ -213,7 +239,8 @@ struct VisitorRowOrScalarBroadcast {
 template<
   class ThreadMap,
   class Element,
-  class StrideMNL = Stride<_1,_0,_0>
+  class StrideMNL = Stride<_1,_0,_0>,
+  bool EnableNullptr = false
```

Comment:
Do we ever exercise the `EnableNullptr = true` case for `VisitorColOrScalarBroadcast`? 

In general, it's best to not write code that's untested and exercised, as untested code should be assumed to be buggy. I suggest keeping the interface as-is but adding a static assert with a comment that the case is unimplemented.

Reply:
Should I remove the changes?